tests/unit_tests/get_model_test.py

```diff
@@ -0,0 +1,25 @@
+import io
+import unittest
+
+from super_gradients.training import models
+from super_gradients.training.utils.config_utils import IS_UNUSED_MESSAGE_INTRO
+
+
+class GetModelTest(unittest.TestCase):
```

Comment:
Please add to suite.

Reply:
I actually moved it to integration test folder, because this is what it actually is